### DISCUSSION TITLE
Dapp如何调用Tron签名支付？

### DISCUSSION THREAD

**Discussion by lz-master** [consenlabs/webview#73] [Q&A]
我现在正在做tron支付项目， 但是文档上没有对tron调用的相关说明， 我尝试着使用TokenWebView.apis.user.showAccountSwitch('TRON')切换到tron链后， 再调用imToken.callAPI(
"transaction.signTransaction",params) 或 window.ethereum.request({
                    method: "eth_sendTransaction",
                    params: [],
                })， 出现《该链下找不到此钱包！》 接下来我应该如何操作？

**Comment by xwartz** [comment]
imToken DApp Browser 注入了 tron provider，支持 tronweb 的签名调用，只要按照 tronweb 调用就行。

https://github.com/tronprotocol/tronweb
